HackIllinois/UI/TableView/Cells/HIBubbleCell/HIEventCell.swift

```diff
@@ -122,10 +122,13 @@ extension HIEventCell {
         let pointsLabel = HILabel(style: .pointsText)
         pointsView.addSubview(pointsLabel)
         pointsLabel.constrain(to: pointsView, topInset: 0, trailingInset: 0, bottomInset: 0, leadingInset: 0)
-        pointsLabel.text = "10 Points!"
+        pointsLabel.text = "\(rhs.points) Points!"
 
         let eventType = HILabel(style: .eventType)
         eventType.text = rhs.eventType
+        if rhs.eventType == "QNA" {
```

Comment:
this should not be special cased in the event cell it should be an extension on eventType which implements CustomStringConvertible